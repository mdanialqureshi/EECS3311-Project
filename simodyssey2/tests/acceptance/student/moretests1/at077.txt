test(1,1,1,1,30)
abort
test(1,1,1,1,30)
abort
test(1,1,1,1,30)
abort
test(1,1,1,1,80)
abort
move(N)
test(1,1,1,1,30)
test(1,1,1,1,30)
test(1,1,1,1,40)
test(1,1,1,1,100)
abort
abort
test(1,1,1,1,100)
move(S)
abort
test(1,1,1,1,4)
move(S)
move(E)
land
status
abort