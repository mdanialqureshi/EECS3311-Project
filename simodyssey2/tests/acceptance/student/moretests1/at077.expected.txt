  state:0.0, ok
  Welcome! Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:1.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   P*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   ----   *---   ----   PW--   
->abort
  state:1.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:2.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   PPP*   PP--   P---   Y---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *---   PY--   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P---   ----   O---   PY--   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   ----   W---   P---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PY--   P---   *---   PW--   
->abort
  state:2.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,30)
  state:3.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EP--   PY--   P---   P---   W---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   ----   ----   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    W---   *---   O---   PPPW   PY--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P*--   W---   P---   P---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   P*--   P---   Y---   
->abort
  state:3.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,80)
  state:4.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[1,P],-,-
    [1,2]->[2,P],[3,P],-,-
    [1,3]->[4,P],-,-,-
    [1,4]->[5,P],-,-,-
    [1,5]->[6,P],[-7,*],-,-
    [2,1]->[7,P],[8,P],-,-
    [2,2]->[9,P],[-9,Y],-,-
    [2,3]->[10,P],[11,P],-,-
    [2,4]->[12,P],-,-,-
    [2,5]->[13,P],[-5,*],-,-
    [3,1]->[-4,W],-,-,-
    [3,2]->[14,P],[15,P],[-11,Y],-
    [3,3]->[-1,O],-,-,-
    [3,4]->[16,P],[17,P],[18,P],-
    [3,5]->-,-,-,-
    [4,1]->[19,P],-,-,-
    [4,2]->-,-,-,-
    [4,3]->[20,P],[21,P],-,-
    [4,4]->[22,P],[-10,Y],-,-
    [4,5]->[23,P],[24,P],[-3,W],-
    [5,1]->[25,P],[26,P],-,-
    [5,2]->[27,P],[-6,Y],-,-
    [5,3]->[28,P],[29,P],[-8,Y],-
    [5,4]->[-2,Y],-,-,-
    [5,5]->[30,P],[31,P],-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,*]->Luminosity:5
    [-6,Y]->Luminosity:2
    [-5,*]->Luminosity:5
    [-4,W]->
    [-3,W]->
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [12,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [17,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [18,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [19,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [20,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [21,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [22,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [23,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [24,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [25,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [26,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [27,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [28,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [30,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EP--   PP--   P---   P---   P*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PP--   PY--   PP--   P---   P*--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    W---   PPY-   O---   PPP-   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   ----   PP--   PY--   PPW-   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PP--   PY--   PPY-   Y---   PP--   
->abort
  state:4.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->move(N)
  state:4.2, error
  Negative on that request:no mission in progress.
->test(1,1,1,1,30)
  state:5.0, mode:test(1,1,1,1,30), ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EP--   P*--   ----   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PW--   ----   W---   ----   *---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   ----   O---   ----   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   P---   ----   W---   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PW--   Y---   P---   PPW-   PY--   
->test(1,1,1,1,30)
  state:5.1, mode:test(1,1,1,1,30), error
  To start a new mission, please abort the current one first.
->test(1,1,1,1,40)
  state:5.2, mode:test(1,1,1,1,30), error
  To start a new mission, please abort the current one first.
->test(1,1,1,1,100)
  state:5.3, mode:test(1,1,1,1,30), error
  To start a new mission, please abort the current one first.
->abort
  state:5.4, ok
  Mission aborted. Try test(1,1,1,1,30)
->abort
  state:5.5, error
  Negative on that request:no mission in progress.
->test(1,1,1,1,100)
  state:6.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[1,P],[2,P],-
    [1,2]->[3,P],[4,P],-,-
    [1,3]->[5,P],[6,P],[7,P],-
    [1,4]->[8,P],[9,P],[10,P],-
    [1,5]->[11,P],[12,P],[-3,W],-
    [2,1]->[13,P],-,-,-
    [2,2]->[14,P],[-9,*],-,-
    [2,3]->[15,P],[-6,Y],-,-
    [2,4]->[16,P],[17,P],[18,P],[-10,*]
    [2,5]->[19,P],[20,P],[21,P],[-4,Y]
    [3,1]->[22,P],[23,P],-,-
    [3,2]->[24,P],[-11,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[25,P],[26,P],[27,P],[-5,Y]
    [3,5]->[28,P],-,-,-
    [4,1]->[29,P],[30,P],-,-
    [4,2]->[31,P],[32,P],-,-
    [4,3]->[33,P],[34,P],[35,P],[-8,*]
    [4,4]->[36,P],[37,P],-,-
    [4,5]->[38,P],-,-,-
    [5,1]->[39,P],-,-,-
    [5,2]->[40,P],[41,P],[42,P],-
    [5,3]->[43,P],[44,P],[45,P],[-7,Y]
    [5,4]->[46,P],[47,P],[48,P],[-2,Y]
    [5,5]->[49,P],-,-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,*]->Luminosity:5
    [-9,*]->Luminosity:5
    [-8,*]->Luminosity:5
    [-7,Y]->Luminosity:2
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,Y]->Luminosity:2
    [-3,W]->
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [7,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [12,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [17,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [18,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [19,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [20,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [21,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [22,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [23,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [24,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [25,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [26,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [27,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [28,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [30,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [32,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [33,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [34,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [35,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [36,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [37,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [38,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [39,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [40,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [41,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [42,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [43,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [44,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [45,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [46,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [47,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [48,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [49,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EPP-   PP--   PPP-   PPP-   PPW-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   P*--   PY--   PPP*   PPPY   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PP--   P*--   O---   PPPY   P---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    PP--   PP--   PPP*   PP--   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   PPP-   PPPY   PPPY   P---   
->move(S)
  state:7.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,2]
    [1,P]:[1,1,2]->[5,1,2]
    [7,P]:[1,3,3]->[2,2,3]
    [8,P]:[1,4,1]->[5,5,2]
    [12,P]:[1,5,2]->[2,1,3]
    [13,P]:[2,1,1]->[3,1,3]
    [22,P]:[3,1,1]->[4,1,3]
    [28,P]:[3,5,1]->[4,5,2]
    [30,P]:[4,1,2]->[4,2,3]
    [49,P]:[5,5,1]->[1,1,1]
  Sectors:
    [1,1]->[49,P],-,[2,P],-
    [1,2]->[3,P],[4,P],-,-
    [1,3]->[5,P],[6,P],-,-
    [1,4]->-,[9,P],[10,P],-
    [1,5]->[11,P],-,[-3,W],-
    [2,1]->-,[0,E],[12,P],-
    [2,2]->[14,P],[-9,*],[7,P],-
    [2,3]->[15,P],[-6,Y],-,-
    [2,4]->[16,P],[17,P],[18,P],[-10,*]
    [2,5]->[19,P],[20,P],[21,P],[-4,Y]
    [3,1]->-,[23,P],[13,P],-
    [3,2]->[24,P],[-11,*],-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->[25,P],[26,P],[27,P],[-5,Y]
    [3,5]->-,-,-,-
    [4,1]->[29,P],-,[22,P],-
    [4,2]->[31,P],[32,P],[30,P],-
    [4,3]->[33,P],[34,P],[35,P],[-8,*]
    [4,4]->[36,P],[37,P],-,-
    [4,5]->[38,P],[28,P],-,-
    [5,1]->[39,P],[1,P],-,-
    [5,2]->[40,P],[41,P],[42,P],-
    [5,3]->[43,P],[44,P],[45,P],[-7,Y]
    [5,4]->[46,P],[47,P],[48,P],[-2,Y]
    [5,5]->-,[8,P],-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,*]->Luminosity:5
    [-9,*]->Luminosity:5
    [-8,*]->Luminosity:5
    [-7,Y]->Luminosity:2
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,Y]->Luminosity:2
    [-3,W]->
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [3,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [5,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [6,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [7,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [9,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [10,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [11,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [12,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [13,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [15,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [17,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [18,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [19,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [20,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [21,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [22,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [23,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [24,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [25,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [26,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [27,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [28,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [30,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [32,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [33,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [34,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [35,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [36,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [37,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [38,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [39,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [40,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [41,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [42,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [43,P]->attached?:T, support_life?:F, visited?:F, turns_left:N/A
    [44,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [45,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [46,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [47,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [48,P]->attached?:T, support_life?:T, visited?:F, turns_left:N/A
    [49,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    P-P-   PP--   PP--   -PP-   P-W-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -EP-   P*P-   PY--   PPP*   PPPY   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -PP-   P*--   O---   PPPY   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P-P-   PPP-   PPP*   PP--   PP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    PP--   PPP-   PPPY   PPPY   -P--   
->abort
  state:7.1, ok
  Mission aborted. Try test(1,1,1,1,30)
->test(1,1,1,1,4)
  state:8.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],-,-,-
    [1,2]->[-11,Y],-,-,-
    [1,3]->[-2,Y],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[-6,*],-,-,-
    [2,1]->-,-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,*],-,-,-
    [2,4]->-,-,-,-
    [2,5]->-,-,-,-
    [3,1]->[-3,*],-,-,-
    [3,2]->[-4,*],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->[-7,*],-,-,-
    [4,2]->[-8,*],-,-,-
    [4,3]->[-9,Y],-,-,-
    [4,4]->-,-,-,-
    [4,5]->[-10,Y],-,-,-
    [5,1]->-,-,-,-
    [5,2]->-,-,-,-
    [5,3]->-,-,-,-
    [5,4]->[1,P],-,-,-
    [5,5]->-,-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,*]->Luminosity:5
    [-5,*]->Luminosity:5
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   Y---   Y---   ----   *---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   *---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    *---   *---   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   *---   Y---   ----   Y---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   ----   P---   ----   
->move(S)
  state:9.0, mode:test, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-11,Y],-,-,-
    [1,3]->[-2,Y],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[-6,*],-,-,-
    [2,1]->[0,E],-,-,-
    [2,2]->-,-,-,-
    [2,3]->[-5,*],-,-,-
    [2,4]->-,-,-,-
    [2,5]->-,-,-,-
    [3,1]->[-3,*],-,-,-
    [3,2]->[-4,*],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->[-7,*],-,-,-
    [4,2]->[-8,*],-,-,-
    [4,3]->[-9,Y],-,-,-
    [4,4]->-,-,-,-
    [4,5]->[-10,Y],-,-,-
    [5,1]->-,-,-,-
    [5,2]->-,-,-,-
    [5,3]->-,-,-,-
    [5,4]->[1,P],-,-,-
    [5,5]->-,-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,*]->Luminosity:5
    [-5,*]->Luminosity:5
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:2/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   Y---   Y---   ----   *---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    E---   ----   *---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    *---   *---   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   *---   Y---   ----   Y---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   ----   P---   ----   
->move(E)
  state:10.0, mode:test, ok
  Movement:
    [0,E]:[2,1,1]->[2,2,1]
    [1,P]:[5,4,1]->[4,4,1]
  Sectors:
    [1,1]->-,-,-,-
    [1,2]->[-11,Y],-,-,-
    [1,3]->[-2,Y],-,-,-
    [1,4]->-,-,-,-
    [1,5]->[-6,*],-,-,-
    [2,1]->-,-,-,-
    [2,2]->[0,E],-,-,-
    [2,3]->[-5,*],-,-,-
    [2,4]->-,-,-,-
    [2,5]->-,-,-,-
    [3,1]->[-3,*],-,-,-
    [3,2]->[-4,*],-,-,-
    [3,3]->[-1,O],-,-,-
    [3,4]->-,-,-,-
    [3,5]->-,-,-,-
    [4,1]->[-7,*],-,-,-
    [4,2]->[-8,*],-,-,-
    [4,3]->[-9,Y],-,-,-
    [4,4]->[1,P],-,-,-
    [4,5]->[-10,Y],-,-,-
    [5,1]->-,-,-,-
    [5,2]->-,-,-,-
    [5,3]->-,-,-,-
    [5,4]->-,-,-,-
    [5,5]->-,-,-,-
  Descriptions:
    [-11,Y]->Luminosity:2
    [-10,Y]->Luminosity:2
    [-9,Y]->Luminosity:2
    [-8,*]->Luminosity:5
    [-7,*]->Luminosity:5
    [-6,*]->Luminosity:5
    [-5,*]->Luminosity:5
    [-4,*]->Luminosity:5
    [-3,*]->Luminosity:5
    [-2,Y]->Luminosity:2
    [-1,O]->
    [0,E]->fuel:1/3, life:3/3, landed?:F
    [1,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   Y---   Y---   ----   *---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   E---   *---   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    *---   *---   O---   ----   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   *---   Y---   P---   Y---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   ----   ----   ----   
->land
  state:10.1, mode:test, error
  Negative on that request:no yellow dwarf at Sector:2:2
->status
  state:10.2, mode:test, ok
  Explorer status report:Travelling at cruise speed at [2,2,1]
  Life units left:3, Fuel units left:1
->abort
  state:10.3, ok
  Mission aborted. Try test(1,1,1,1,30)
